src/cpuid/bindings.pyi

```diff
@@ -1,4 +1,5 @@
-from typing import Tuple
+def cpuid(level: int) -> tuple[int, int, int, int]:
```

Comment:
```suggestion
def cpuid(level: int) -> Tuple[int, int, int, int]:
```

Use the type from the typing module, this is breaking for older versions of Python.

Reply:
You're right, done in [08145e0](https://github.com/bearmetaldev/python-cpuid/pull/8/commits/08145e064176bbfdfedf16aa60902d51826f308f).